jaxlib/init.py

```diff
@@ -12,4 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+def include_dir() -> str:
```

Comment:
I think we should put this Python API in *jax*. In general jaxlib has no public APIs and I'd like to keep it that way.

This function is "where do I find the headers" and it can point to jaxlib. But it should be a jax API so we can change the package structure later.